src/lib/transport.js

```diff
@@ -203,6 +204,15 @@ Transport.prototype.request = function (params, cb) {
     requestTimeout = params.requestTimeout;
   }
 
+  const pingRequest = params.path === '/' && params.method === 'HEAD';
+  if (pingRequest) {
+    const pingParam =
```

Comment:
nit: I don't think it's necessary to split these lines up and IMO makes it easier to read

```
const pingParam = params.hasOwnProperty('pingTimeout') && params.pingTimeout;
const requestParam = params.hasOwnProperty('requestTimeout') && params.requestTimeout;

requestTimeout = pingParam || requestParam || this.pingTimeout || requestTimeout;
```

Reply:
👍 that was an autofix done by prettier, but it's not enabled here so i'll revert